frontend/app/src/scenes/Home/scenes/UpsertCategory/index.tsx

```diff
@@ -139,7 +203,92 @@ class UpsertCategory extends React.Component<
                       >
                         <Input placeholder="ex: KG & Primary" />
                       </Form.Item>
-                      <Form.Item>
+                      <Title level={3}>Translations</Title>
+                      {Array.from(this.state.translations.keys()).map(
```

Comment:
you have used `Array.from(this.state.translations.entries()).map` at line 88. 
can we use the same here? I'm not sure what is the optimized one. Do the research and use the best.

Reply:
sure, I'll look into it.

Reply:
seems like entries better we don’t have to do the null check and we can remove the language variable

Reply:
great.